core/src/test/java/org/elasticsearch/indices/IndexingMemoryControllerTests.java

```diff
@@ -181,7 +181,7 @@ public void testShardAdditionAndRemoval() {
         IndexService test = indicesService.indexService(resolveIndex("test"));
 
         MockController controller = new MockController(Settings.builder()
-                .put(IndexingMemoryController.INDEX_BUFFER_SIZE_SETTING, "4mb").build());
+                                                       .put(IndexingMemoryController.INDEX_BUFFER_SIZE_SETTING.getKey(), "4mb").build());
```

Comment:
I like that!  I'll fix the cases in IMCTests.java here ... probably we should make such constants (e.g. the ones in IMC.java) private in the future.